presto-iceberg/src/main/java/com/facebook/presto/iceberg/util/IcebergStatisticsUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg.util;
+
+import com.facebook.presto.common.type.Type;
+import com.facebook.presto.common.type.TypeUtils;
+import com.facebook.presto.spi.statistics.ColumnStatisticMetadata;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+import static com.facebook.presto.common.type.Chars.isCharType;
+import static com.facebook.presto.common.type.DateType.DATE;
+import static com.facebook.presto.common.type.TimestampType.TIMESTAMP;
+import static com.facebook.presto.common.type.Varchars.isVarcharType;
+import static com.facebook.presto.spi.statistics.ColumnStatisticType.NUMBER_OF_DISTINCT_VALUES;
+
+public class IcebergStatisticsUtil
+{
+    private IcebergStatisticsUtil()
+    {
+    }
+
+    public static List<ColumnStatisticMetadata> getSupportedColumnStatistics(String columnName, Type type)
```

Comment:
I figure in the future more methods could be added. However, since this is the only usage, I might as well just leave it in the TableStatisticsMaker class